src/pci_lmt/pcie_lane_margining.py

```diff
@@ -718,14 +718,6 @@ def step_margin_voltage_offset_up_down_of_default(
         if receiver_number not in [*range(0x1, 0x7)]:
```

Comment:
whenever there's bare numbers like this, i suggest naming them or putting in a comment with a reference to the meaning of said number